docs/get-started/using-with-typescript.md

```diff
@@ -1,33 +1,23 @@
 # Using deck.gl with TypeScript
 
-Starting from v8.8, deck.gl publishes "public preview" TypeScript types via an opt-in only entry point. To use the official types in your application, find the following statements:
+Starting from v9.0, deck.gl publishes official TypeScript types for all modules. If TypeScript is enabled in your project, imports from deck.gl packages will include types. Examples:
```

Comment:
I would keep the old text and move it under a section "Using deck.gl v8 with TypeScript".

Reply:
Done!